lib/vfscore/file.c

```diff
@@ -36,9 +36,9 @@
 #include <errno.h>
 #include <uk/print.h>
 #include <vfscore/file.h>
-#include <vfscore/eventpoll.h>
 #include <uk/assert.h>
 #include "vfs.h"
+#include <vfscore/eventpoll.h>
```

Comment:
Could try out:
```C
#include "vfs.h"

#include <unistd.h>
#include <stdlib.h>
#include <errno.h>
#include <uk/print.h>
#include <uk/assert.h>
#include <vfscore/file.h>
#include <vfscore/eventpoll.h>
```
This is cleaner as it moves the internal header up instead of some (random) dependency down.